binding/web/test/index.html

```diff
@@ -6,25 +6,26 @@
   <script src="../dist/iife/index.js"></script>
   <script src="./leopard_params.js"></script>
   <script type="application/javascript">
-    const expected = "Mr. Quilter is the apostle of the middle classes and we are glad to welcome his gospel.";
+    const expected = "Mr quilter is the apostle of the middle classes and we are glad to welcome his gospel";
+    const expectedPunctuation = "Mr. Quilter is the apostle of the middle classes and we are glad to welcome his gospel.";
     const expectedWords = [
-      {word: 'Mr'       , startSec: 0.576, endSec: 0.800},
-      {word: 'quilter'  , startSec: 0.864, endSec: 1.184},
-      {word: 'is'       , startSec: 1.312, endSec: 1.376},
-      {word: 'the'      , startSec: 1.440, endSec: 1.503},
-      {word: 'apostle'  , startSec: 1.568, endSec: 2.080},
-      {word: 'of'       , startSec: 2.176, endSec: 2.240},
-      {word: 'the'      , startSec: 2.304, endSec: 2.336},
-      {word: 'middle'   , startSec: 2.400, endSec: 2.592},
-      {word: 'classes'  , startSec: 2.688, endSec: 3.168},
-      {word: 'and'      , startSec: 3.360, endSec: 3.456},
-      {word: 'we'       , startSec: 3.520, endSec: 3.552},
-      {word: 'are'      , startSec: 3.648, endSec: 3.648},
-      {word: 'glad'     , startSec: 3.744, endSec: 4.032},
-      {word: 'to'       , startSec: 4.096, endSec: 4.160},
-      {word: 'welcome'  , startSec: 4.223, endSec: 4.576},
-      {word: 'his'      , startSec: 4.672, endSec: 4.832},
-      {word: 'gospel'   , startSec: 4.928, endSec: 5.376},
```

Comment:
these confidence levels will change later and `DeepEqual` will fail. So i guess we can only check if there are higher that some threshold